monitoring/repo_updater.go

```diff
@@ -154,21 +156,22 @@ func RepoUpdater() *Container {
 							PossibleSolutions: "Check repo-updater logs. This is expected to fire if there are no user added code hosts",
 						},
 						Observable{
-							Name:              "sched_update_queue_length",
-							Description:       "repositories queued for update",
-							Query:             `sum(src_repoupdater_sched_update_queue_length)`,
-							DataMayNotExist:   true,
-							Critical:          Alert().GreaterOrEqual(1000).For(5 * time.Minute),
+							Name:            "sched_update_queue_length_derivative",
```

Comment:
```suggestion
							Name:            "sched_update_queue_length",
```

I think we can keep the alert names the same, since in principle you are alerting on the same thing (behaviour of length of this queue)